kubernetes/launch-job.sh

```diff
@@ -7,7 +7,8 @@
 DEFAULT_NUM_VICTIMPLAY_GPUS=4
 
 usage() {
-  echo "Usage: $0 [--victimplay-gpus GPUS] [--use-weka] PREFIX"
+  echo "Usage: $0 [--victimplay-gpus GPUS] [--victimplay-max-gpus MAX_GPUS]"
+  echo "         [--resume TIMESTAMP] [--use-weka] PREFIX"
```

Comment:
Align it with above
```suggestion
  echo "          [--resume TIMESTAMP] [--use-weka] PREFIX"
```

Reply:
that does look visually nicer in the file but it doesn't look nicer (or worse) when actually printed — `$0` is replaced with `./launch-job.sh` so the aligning is lost anyway